.changes/backported/BUG FIXES-20250122-083138.yaml

```
@@ -0,0 +1,5 @@
+kind: BUG FIXES
+body: Updated dependency github.com/hashicorp/go-slug v0.16.0 => v0.16.3 to integrate latest changes (fix for CVE-2025-0377)
```

```suggestion
body: Updated dependency `github.com/hashicorp/go-slug` `v0.16.0` => `v0.16.3` to integrate latest changes (fix for CVE-2025-0377)
```

I'm in favour of keeping the backticks for a more readable rendered markdown